### DISCUSSION TITLE
Change shutter speed in time lapse photo on Hero10

### DISCUSSION THREAD

**Discussion by m-h-williams** [gopro/labs#625] [Q&A] [answered]
Hello! 

I am hoping to use time lapse photo mode in a GoPro Hero 10 to take a photo every second at 30fps, but I want the shutter speed to be 1/20 or 1/60 to reduce blur as we move. I see you can increase the shutter speed for night photos, but is it possible to reduce the shutter speed of a time lapse photo? 

Thanks for your help!

**Comment by dnewman-gpsw** [comment, marked as answer]
1/20 or 1/60 is still some amount of blur, and likely too much light in daylight without ND filters.  There was a control added for HERO11 and 12 Labs, e.g. with this command: oMEXPX=20oMEXPN=60, the maximum exposure time is 1/20 and the minimum is 1/60.  

There might be a Labs update for HERO10 in the future (not soon), if so this feature has been added to the request features list.

**Reply by m-h-williams** [reply]
Thank you! So is 1/60th the absolute maximum for the shutter speed? Or can that be brought down further in HERO11?

**Reply by dnewman-gpsw** [reply]
No, it can range approximately from 1/4 to 1/4000

**Reply by m-h-williams** [reply]
Wow! Should I put in a feature request for HERO10, or are you saying there already is one?

**Reply by dnewman-gpsw** [reply]
Feature request is on the list.
